### PR TITLE
Add five new keyboard shortcuts:

### PR DESCRIPTION
**Pull request by mirabilos**
* `#` `=` `-`	to add ♯ ♮ ♭ respectively
* `(`	to add parenthesēs around the note/accidental
* `)`	to exclude the current note from beaming

### CODE REVIEW

**Comment by MarcSabatella** [comment]
I missed the discussion that led to this.  I definitely like the idea of having shortcuts for accidentals but am not sure these in particular are safe on all keyboard layouts.  Also I'm not comfortable with using ( and ) for things that are not related to each other - they seem potentially more valuable for other purposes.  And if there is to be a default shortcut for "no beam" I think there needs to be related ones for "start of beam" and "middle of beam".

Probably all of this should be discussed on the forum to get more feedback from users.

**Comment by mirabilos** [comment]
There was no discussion beforehand — this is simply an attempt to push shortcuts for keys that ① are not bound before and ② really help with speedy note input upstream. (These are the only divergence from MuseScore standard keybindings I have.) All in the sense of being a good user-cum-poco-developer. (Actually got this idea while replying to something from, I think, @lasconic on Twitter.)

I hope the use of `#`, `=` and `-` is not disputed.

As for `(`, this is the _only_ really mnemonic shortcut for putting parenthesēs around a note or accidental. It does not have a “closing” match, not does anything else fit for `)`, so why not “no beam”?

Rationale for “no beam”: When entering notes, I’ve never **ever** had to add extra beaming to what MuseScore did by default (from the TimeSig), only to disable beaming for some notes.

Rationale for using `)` for “no beam”: I thought of binding it elsewhere, but since `(` was already used by something that did not have any symmetry (“no beam” is my most recent addition) it was available IMHO. And mnemonically it also fits: it “excludes” (put outside the parenthesised group) the note from beaming.

Regarding users: As this will be a mere addition to the default shortcuts, only new users will get it by default, and any old users who already use either of the five keys for anything else will keep their old bindings. As such, it should not, no, *cannot* get into the way of someone.

In conclusion, I hope these five new shortcuts provide useful — I might be biased but I think it’s *clearly* better than not having them.

**Comment by MarcSabatella** [comment]
As I said, I think it should be discussed in the forum first to get input on which shortcuts are most useful and most system-independent.  Other people may have valid opinion on which commands are most important as well as which specific shortcuts make the most sense.  It's just how we normally do things here - first gather input, then after a consensus is reached, go ahead and implement.

I've already expressed my reservation with using ) for no beam and providing nothing for the other beam commands.  I could make an alternate suggestion,. but again, it's the sort of thing best discussed in the forum, in my opinion.

**Comment by mirabilos** [comment]
I was hoping developer review would suffice, but here you are:

https://musescore.org/en/node/264557

**Comment by lasconic** [comment]
OK. It looks the consensus is that it's better to not have default shortcuts for most of these actions. I will close this PR. Please reopen if consensus would change.

**Comment by mirabilos** [comment]
Looks more like any consensus is hard to reach, but I agree to close this.

For the record (in case anyone else stumbles upon this), http://mirsolutions.de/music/ now contains a `shortcuts.xml` file implementing this (currently for 2.1, ought to be usable with 2.0.3 and 2.2~nightly). I’ll keep maintaining this independently, then.
